react/components/ProductSummaryBuyButton/ProductSummaryBuyButton.js

```diff
@@ -73,37 +75,22 @@ const ProductSummaryBuyButton = ({
   // if the item is not available the behavior is just show the disabled BuyButton,
   // but you still can go to the product page clicking in the summary
   const shouldBeALink =
-    (items.length !== 1 || buyButtonBehavior !== BUY_BUTTON_BEHAVIOR_OPTIONS) &&
+    (items.length !== 1 || buyButtonBehavior !== DEFAULT_BUTTON_BEHAVIOR) &&
     isAvailable
 
   return (
     showBuyButton && (
       <div className={containerClass}>
         <div className={buyButtonClasses}>
-          {shouldBeALink ? (
-            <Link
-              className="dib"
-              disabled
-              page="store.product"
-              params={{
-                slug: product && product.linkText,
-                id: product && product.productId,
-              }}
-            >
-              <Button>
-                <IOMessage id={buyButtonText} />
-              </Button>
-            </Link>
```

Comment:
SHould this really be removed? Won't this remove the feature you made previously?

Reply:
Yes and no, because the code was buggy, if you check the new code has the same functionality that I made previeously